### PR TITLE
Add supporters page copy to the site

### LINKED ISSUE #17: VVV/E Add Supporter Page to seAp site
##  Description

We need to add the supporters page copy to the site

Link to the copy: https://docs.google.com/spreadsheets/d/1aFDhogqKhC2AogDuwdOVTlOytb3NxbsYjOaka43ssbw/edit?usp=sharing

## Plan

- [x] @roseb04 to get sign off on copy
- [x] Add copy
- [x] Add correct text for **`FooterCopyrightLink`** section - [see comment below](https://github.com/neontribe/ca-seap/issues/17#issuecomment-566582839)

## Acceptance Criteria

- [x] Copy matched the copy in the copy spreadsheet 
- [x] Supporters is selectable from home page

### PR DESCRIPTION
**Pull request by AlinChindea**
Fixes #17 

## Description

- added content from copy

@neontribe/ca-seap

### CODE REVIEW

File: src/Copy/BrandCopy.elm [hunk 1]
```diff
@@ -282,37 +282,37 @@ brandCopy key =
...
-            CopyText "This page, available in the landing page footer, allows organisations to thank their partners."
+            CopyText "We would like to thank our funders, whose generosity has made this website possible."
 
         SupportersP2 ->
-            CopyText "We are grateful to all the partners that have made this project possible."
+            CopyText "This web app was funded by CAST - Centre for Acceleration of Social Technology - as part of a project called Contemplating Action which is seeking to find out whether visual stories can help people overcome their fears and barriers to getting support. The project has produced an app that could be used by different charities to encourage people to reach out and get help by reassuring them that they are not alone."
 
         CastLogoAlt ->
-            CopyText "CAST"
+            CopyText "Centre for accelerating social technology logo"
```
Comment (katjam) [inline src/Copy/BrandCopy.elm]: Wording differs from line 288

File: src/Copy/BrandCopy.elm [hunk 2]
```diff
@@ -282,37 +282,37 @@ brandCopy key =
...
-            CopyText "Neontribe"
+            CopyText "Neontribe logo"
 
         HavenLogoAlt ->
-            CopyText "The Haven Wolverhampton"
+            CopyText "The Haven Wolverhampton logo"
 
         SupportersP5 ->
-            CopyText "This website has been developed by Neontribe and The Haven, together with women who have been victims of domestic abuse. The Haven is a Wolverhampton based charity which supports women and dependent children who are vulnerable to domestic violence, homelessness and abuse."
+            CopyText "This web app was built by Neontribe and seAp. This website has been developed by Neontribe and seAp, together with people who have used seAp’s health care complaints service."
```
Comment (katjam) [inline src/Copy/BrandCopy.elm]: Copy a little repetitive and confusing web app then website.

**Review by katjam — COMMENTED** [review]
I'd like @roseb04 to review copy before merging.

File: src/Copy/BrandCopy.elm (hunk 1 above)
Comment (rosejbon) [reply src/Copy/BrandCopy.elm]: Good spot, updated accordingly

File: src/Copy/BrandCopy.elm (hunk 2 above)
Comment (rosejbon) [reply src/Copy/BrandCopy.elm]: Good point, I've amended the copy to be a bit more concise and less repetitive

**Comment by rosejbon** [comment]
@katjam and @AlinChindea I'm happy with this branch, good to merge when your ready

File: src/Copy/BrandCopy.elm [hunk 3]
```diff
@@ -279,40 +279,40 @@ brandCopy key =
...
 
         HavenLogoAlt ->
-            CopyText "✨"
+            CopyText "The Haven Wolverhampton logo"
 
         SupportersP5 ->
-            CopyText "✨"
+            CopyText "This web app was built by Neontribe and seAp, together with people who have used seAp’s health care complaints service."
 
         FooterSupportersText ->
```
Comment (katjam) [inline src/Copy/BrandCopy.elm:317]: We should put the defaults back in here. Might as well with this PR.

Comment (AlinChindea) [reply src/Copy/BrandCopy.elm:317]: @katjam added. can we merge?

Comment (katjam) [reply src/Copy/BrandCopy.elm:317]: @AlinChindea Go for it.

**Pull request merged**